ts/screens/onboarding/EmailValidateScreen.tsx

```diff
@@ -0,0 +1,134 @@
+
+type NavigationParams = {
+  biometryType: BiometrySimpleType;
+};
+
+type OwnProps = ReduxProps &
+  ReturnType<typeof mapStateToProps> &
+  NavigationScreenProps<NavigationParams>;
+
+type Props = OwnProps & ReturnType<typeof mapDispatchToProps>;
+
+const styles = StyleSheet.create({
+  spacerLarge: { height: 40 }
+});
+
+const unavailableAlert = () => Alert.alert(I18n.t("global.notImplemented"));
+
+/**
+ * A screen to show if the fingerprint is supported to the user.
+ */
+export class FingerprintScreen extends React.PureComponent<Props> {
+  private handleGoBack = () =>
+    Alert.alert(
+      I18n.t("onboarding.alert.title"),
+      I18n.t("onboarding.alert.description"),
+      [
+        {
+          text: I18n.t("global.buttons.cancel"),
+          style: "cancel"
+        },
+        {
+          text: I18n.t("global.buttons.exit"),
+          style: "default",
+          onPress: () => this.props.abortOnboarding()
+        }
+      ]
+    );
+
+  public render() {
+    const { potProfile } = this.props;
+
+    const profileData = potProfile
+      .map(_ => ({
+        spid_email: untag(_.spid_email)
+      }))
+      .getOrElse({
+        spid_email: ""
+      });
+
+    return (
+      <TopScreenComponent
+        goBack={this.handleGoBack}
+        headerTitle={I18n.t("onboarding.email.validation.headerTitle")}
+        title={I18n.t("onboarding.email.validation.title")}
+        contextualHelp={{
+          title: I18n.t("onboarding.email.validation.title"),
+          body: () => (
+            <Markdown>{I18n.t("onboarding.email.validation.help")}</Markdown>
+          )
+        }}
+      >
+        <ScreenContent title={I18n.t("onboarding.email.validation.title")}>
+          <View content={true}>
+            <Markdown>
+              {I18n.t("onboarding.email.validation.info", {
+                email: profileData.spid_email
+              })}
+            </Markdown>
+            <View style={styles.spacerLarge} />
+            <View>
+              <Button
+                light={true}
+                block={true}
+                bordered={true}
+                onPress={unavailableAlert}
+              >
+                <Text>{I18n.t("onboarding.email.validation.ctaValidate")}</Text>
+              </Button>
+            </View>
+          </View>
+        </ScreenContent>
+        <FooterWithButtons
+          type={"TwoButtonsInlineHalf"}
+          leftButton={{
+            block: true,
+            bordered: true,
+            title: I18n.t("onboarding.email.ctaEdit"),
+            onPress: unavailableAlert
+          }}
+          rightButton={{
+            block: true,
+            primary: true,
+            title: I18n.t("global.buttons.continue"),
+            onPress: this.props.acknowledgeEmail
+          }}
+        />
+      </TopScreenComponent>
+    );
+  }
+}
+
+const mapStateToProps = (state: GlobalState) => ({
+  potProfile: pot.toOption(state.profile)
```

Comment:
same of here https://github.com/teamdigitale/io-app/pull/1308#discussion_r327025800

Reply:
changed in cdb554f8